desktop/cmp/grid/impl/ColChooser.js

```diff
@@ -32,6 +33,10 @@ export class ColChooser extends Component {
 
     static modelClass = ColChooserModel;
 
+    static propTypes = {
+        model: PT.oneOfType([PT.instanceOf(ColChooserModel), PT.object]).isRequired
+    };
```

Comment:
Pls. remove, as above re. implementation classes.  In this case, `Grid` always renders this component and passes it a fully-formed model.